src/app/utils/LocalStorage.ts

```diff
@@ -37,6 +37,7 @@
  */
 
 export enum LocalStorageKey {
+  AUTOMATED_ANALYSIS_FILTERS,
```

Comment:
Since now we have more keys, could you remove this line here?

https://github.com/cryostatio/cryostat-web/blob/e8a1ebc6cc7877c76827c596f551c5057987eb7a/src/app/utils/LocalStorage.ts#L48